app/lib/controlled_work_mappings/cw2_header.yml

```diff
@@ -0,0 +1,261 @@
+# This file defines the mappings used to prepopulate a CW2 IMM PDF form with the answers given by a user to CCQ's questions
```

Comment:
This alarms me - GitHub hasn't picked up that this file is just `cw2_mtr_phase_2.yml` renamed, although it has done for every other new yaml file. Are you confident that this is identical to the file you've deleted?

Reply:
I can explain - it was my 'neatness' itch, that I had to scratch. 

This **was** copied from the previous `cw2_mtr_phase_1.yml` file. I just indented mappings to the left, to match all the other mappings.